fastlane/sync_mock_server.py

```diff
@@ -0,0 +1,207 @@
+import aiohttp
+import asyncio
+import json
+import uuid
+import os
```

Comment:
Any reason why we are doing this with AIOHTTP and not, for example, Sinatra, since we are using Sinatra for other things? My concern is that we might be adding too much tech here, were we could just stick to ruby 🤔

Reply:
Eh, to be honest, I'd love to, we use Ruby for almost everything around building and scripting (fastlane + sinatra), but for this particular task I just could not come up with something on Ruby - I've searched for any possible solution on Ruby to work with WebSockets, but it did not work out with any of them. Unfortunately, most of them are quite old and not supported. With Python, it just worked like a charm from the first try.